src/controller/artifact/controller.go

```diff
@@ -56,6 +56,8 @@ import (
 	model_tag "github.com/goharbor/harbor/src/pkg/tag/model/tag"
 )
 
+const inTotoContentType = "application/vnd.in-toto+json"
```

Comment:
can we define an array at least? So then next time, we could just modify data.

Reply:
Updated